configure.ac

```diff
@@ -25,6 +28,16 @@ AC_CHECK_PROGS([ANT], [ant])
 
 PKG_CHECK_MODULES([json_c], [json-c])
 
+if test "x${enable_hardening}" != "xno"; then
+  AS_VAR_APPEND([CFLAGS], [" -Qunused-arguments -fstack-protector-all -fPIE -Wa,--noexecstack -Wformat -Wformat-security"])
+  AS_VAR_APPEND([CPPFLAGS], [" -D_FORTIFY_SOURCE=2"])
```

Comment:
I'd move this line up by one, to have to two "related" `CFLAGS` and `CXXFLAGS` together?